composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "igalia/phpreport",
     "description": "phpreport",
     "type": "project",
-    "license": "GNU GPLv3",
+    "license": "LGPL-3.0-or-later",
```

Comment:
There are a few imported lines of LGPL code, but the whole application is distributed under GPL terms. Is there a problem with "GPL-3.0-or-later"?

Reply:
No problem, I've just changed it to GPL-3.0-or-later